### DISCUSSION TITLE
Taskbar and desktop broke

### DISCUSSION THREAD

**Discussion by dougy2shoez** [valinet/ExplorerPatcher#873] [General]
When I ran the program, My taskbar disappeared and my desktop is broken! I tried restarting explorer but it didn't work.
The program didn't give me a wizard or anything.

**Comment by valinet** [comment]
Wizard to do what? The idea is you double click it, it install and just works. That is sometimes fails to do something, yeah, it’s possible, it has to be investigated and fixed.

**Reply by dougy2shoez** [reply]
or like i don't know what it was supposed to open, cuz it didnt open anything and my taskbar is gone

**Reply by dougy2shoez** [reply]
and i cant send you a screenshot bc its not allowing me to open some 
programs with shortcuts

**Reply by dougy2shoez** [reply]
theres also no uninstaller

**Reply by valinet** [reply]
There is. Check the README file.

**Reply by dougy2shoez** [reply]
yeah that did nothing

**Reply by dougy2shoez** [reply]
uninstalling)

**Reply by dougy2shoez** [reply]
do i reboot?
^^^

sorry

**Reply by valinet** [reply]
Write 1 message with everything you want to ask. This is neither a chat, nor a live support board.

**Reply by dougy2shoez** [reply]
![image](https://user-images.githubusercontent.com/79595919/154316399-79a25d8a-3375-458e-9a97-e7fdeff63fcc.png)
Instead of being a small program icon, its a wide program icon, and I cant seem to find a way to disable that.

**Reply by valinet** [reply]
The articles on the wiki are meant to be read and are there to help: https://github.com/valinet/ExplorerPatcher/wiki/All-features. That was my hope with them, now I feel like I just wasted my time. They have no purpose if they are ignored almost all of the times. By reading the wiki, you may find answers to a lot of common questions that a lot of you prefer to repost again and again and again.
